DVR/Session.swift

```diff
@@ -214,9 +220,9 @@ public class Session: NSURLSession {
             if let data = string.dataUsingEncoding(NSUTF8StringEncoding) {
                 data.writeToFile(outputPath, atomically: true)
                 print("[DVR] Persisted cassette at \(outputPath). Please add this file to your test target")
+            } else {
+                print("[DVR] Failed to persist cassette.")
```

Comment:
In my PR to convert to Swift, I added a `return` on 223 instead. Slightly more concise :)